rechnung/tests/fixtures/payments/postbank.yaml

```diff
@@ -0,0 +1,32 @@
+- amount: !!python/object/apply:decimal.Decimal
```

Comment:
I'm again storing Python objects in Yaml as we can't use the yaml safe load mode anymore...

Reply:
Yeah. That's a good point.